test/authlete/model/test_client.rb

```diff
@@ -115,6 +115,7 @@ class ClientTest < Minitest::Test
   ATTRIBUTE_KEY                              = '<attribute0-key>'
   ATTRIBUTE_VALUE                            = '<attribute0-value>'
   ATTRIBUTES                                 = [ Authlete::Model::Pair.new(key: ATTRIBUTE_KEY, value: ATTRIBUTE_VALUE) ]
+  CUSTOM_METADATA                            = 'JSON string'
```

Comment:
Can you repalce the `'JSON String'` with `'<custom-matadata>'` ?

Reply:
I have made the change.